examples/adjoints/adjoints_ex7/SigmaPhysics.h

```diff
@@ -0,0 +1,55 @@
+#include "libmesh/fem_system.h"
+#include "libmesh/mesh.h"
+#include "libmesh/mesh_function.h"
+#include "libmesh/fem_physics.h"
+#include "libmesh/system.h"
+
+// Using namespace libMesh
+using namespace libMesh;
+
+// FEMSystem, TimeSolver and  NewtonSolver will handle most tasks,
+// but we must specify element residuals
+class SigmaPhysics : public FEMPhysics
+{
+public:
+  // Constructor
+  SigmaPhysics():FEMPhysics(){}
+
+  std::string & fe_family() { return _fe_family; }
+  unsigned int & fe_order() { return _fe_order; }
```

Comment:
Are these accessors or the variables they access getting used anywhere?  They look like they're copy-and-paste cruft from a System subclass where they would have been used.

Reply:
Yeah, all this excess needs to be trimmed.